recipes/heavydb/meta.yaml

```
@@ -0,0 +1,575 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "0" %}
```

Given that omniscidb-feedstock provides heavydb v 6.0.0 packages, I suggest temporarily using a non-zero build number:
```suggestion
{% set number = "3" %}
```
to avoid possible heavydb package conflicts.

---

Good idea